instrumentkind_string.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by "stringer -type=InstrumentKind"; DO NOT EDIT.
 
-package otel
+package otel // import "go.opentelemetry.io/otel"
```

Comment:
Modifying these without also rebuilding the `stringer` utility is going to always cause conflict. We should excluded these generated files.